modules/ROOT/pages/to-request-access.adoc

```diff
@@ -26,19 +26,19 @@ If terms and conditions are set for an asset, you are prompted to accept the
 terms before access is granted, and the created contract includes the terms and
 conditions.
 
-. Click an API asset to view more details.
+. Click an API or API Group asset to view more details.
 . Click *Request Access*.
-. Choose the API instance from the list.
+. Choose the API instance or API Group instance from the list.
```

Comment:
Maybe we can say (to avoid redundancy):
Choose the desired instance from the list